app/services/publishing_api_workflow_bypass_publisher.rb

```diff
@@ -0,0 +1,56 @@
+class PublishingApiWorkflowBypassPublisher
+  def initialize(artefact)
+    @artefact = artefact
+  end
+
+  def self.call(artefact)
+    new(artefact).call
+  end
+
+  def call
+    return if artefact.nil?
+
+    discard_draft if has_local_draft?
+    republish_currently_live_edition if has_live_edition?
+    put_draft_content if has_local_draft?
+  end
+
+private
+
+  attr_reader :artefact
+
+  def has_local_draft?
+    draft_edition.present?
+  end
+
+  def has_live_edition?
+    live_edition.present?
+  end
+
+  def discard_draft
+    Services.publishing_api.discard_draft(artefact.content_id)
+  end
+
+  def republish_currently_live_edition
+    PublishingAPIUpdater.new.perform(live_edition.id)
+    PublishingAPIPublisher.new.perform(live_edition.id)
```

Comment:
Should this be `PublishingAPIPublisher.new.perform(edition_id, 'republish')` instead? Really not sure on this but wondering if a 'normal' publish would trigger some sort of commpunication/emails/fanfares somewhere?